pkg/executor/plugins/docker/plugin/plugin.go

```diff
@@ -0,0 +1,81 @@
+package main
```

Comment:
This is the pluggable executor, right now it is wrapping the docker executor. Later implementation will need to satisfy the `PluggableExecutor` interface (as this is doing) and implement a similar `main` function to be pluggable to a compute node.